brush/src/components/ObjectsTable.js

```diff
@@ -0,0 +1,73 @@
+import React, { useContext } from 'react'
+import PropTypes from 'prop-types'
+import { Table } from 'former-kit'
+import scissors from 'scissors'
+import { defaultTo } from 'ramda'
+import VisionContext from '../context/VisionContext'
+
+const ObjectsTable = ({ onRowClickHandle }) => {
+  const { oam } = useContext(VisionContext).vision
```

Comment:
What means `oam`  ?

Reply:
`oam` is [a concept in GBA](https://www.coranac.com/tonc/text/objbg.htm). OAM = Object Attribute Memory.
That is, a region in memory that store the objects informations shown at the screen. At the Klonoa's game, also has a "global oam", that store the all objects informations at the level. For example, a coin shown in Mario's game is an object.

I really don't know a good name to use in this case, because "OAM" refers to the memory region, and "object" is the data stored in this place... but JS already has things that is called "object"...

Reply:
@Deivis `oam` is [a concept in GBA](https://www.coranac.com/tonc/text/objbg.htm). OAM = Object Attribute Memory.
That is, a region in memory that store the objects informations shown at the screen. At the Klonoa's game, also has a "global oam", that store the all objects informations at the level. For example, a coin shown in Mario's game is an object.

I really don't know a good name to use in this case, because "OAM" refers to the memory region, and "object" is the data stored in this place... but JS already has things that is called "object"...